metrics/processors/node_aggregator.go

```diff
@@ -42,8 +42,7 @@ func (this *NodeAggregator) Process(batch *core.DataBatch) (*core.DataBatch, err
 				node, found := batch.MetricSets[nodeKey]
 				if !found {
 					glog.Warningf("Failed to find node: %s", nodeKey)
```

Comment:
I meant continue as in continue with the for loop. Reads better to me but only stylistic preference.

I'm still unsure why changing the resolution makes it likely to hit this condition. Think that needs looking at more.